### ISSUE TITLE
Intel Arc B580 is not detected although LibreHardwareMonitor detects it correctly

### ISSUE THREAD

**Issue by diegocoliveira** [thewriteway/SidebarDiagnostics#88]
### Environment

- Sidebar Diagnostics: v3.7.4
- Windows 11 25H2 (Build 26200.8737)
- CPU: AMD Ryzen 5 5600GT
- GPU: Intel Arc B580 12 GB (Battlemage)
- Intel Graphics Driver: 32.0.101.8861

### Description

Sidebar Diagnostics does not detect my Intel Arc B580.

CPU, RAM, Drives and Network monitors work correctly, however the GPU monitor cannot detect any compatible hardware.

Inside **Settings → Monitors → GPU**, the **Hardware** list is completely empty, making it impossible to select any GPU metrics.

### Expected behavior

The Intel Arc B580 should appear in the GPU Hardware list and expose available sensors such as:

- GPU Temperature
- GPU Load
- GPU Package Power
- GPU Core Clock
- GPU Memory Clock
- GPU Fan Speed
- VRAM Usage

### Actual behavior

No Intel GPU appears in the Hardware list.

### Troubleshooting already performed

- Updated Intel graphics driver
- Updated motherboard BIOS
- Disabled integrated GPU (IGFX Multi-Monitor) in BIOS
- Clean driver installation
- Windows correctly detects the Intel Arc B580
- Intel Graphics Software correctly detects the GPU
- GPU-Z correctly detects all GPU sensors

### LibreHardwareMonitor verification

To isolate the problem, I downloaded the latest official LibreHardwareMonitor release.

LibreHardwareMonitor detects the Intel Arc B580 correctly and exposes all expected sensors, including:

- GPU Temperature
- GPU Core Voltage
- GPU Package Power
- GPU Core Clock
- GPU Memory Clock
- GPU Load
- GPU Render / Compute
- GPU Memory Usage
- GPU Fan Speed
- VRAM Total / Used / Free

This strongly suggests:

- the hardware is working correctly;
- the Intel driver is working correctly;
- LibreHardwareMonitor itself supports the Intel Arc B580 correctly.

### Additional test

I also attempted to replace Sidebar Diagnostics' `LibreHardwareMonitorLib.dll` with the official DLL from the latest LibreHardwareMonitor release.

Sidebar then failed to start with:

```
System.MissingMethodException
Method not found...
```

which indicates Sidebar was compiled against a different revision of LibreHardwareMonitor.

### Screenshots attached

1. Sidebar Diagnostics showing an empty GPU Hardware list.
2. LibreHardwareMonitor correctly detecting the Intel Arc B580 and all sensors.
3. GPU-Z showing all GPU telemetry.
4. MissingMethodException after replacing LibreHardwareMonitorLib.dll.

I would be happy to test preview builds or provide additional logs if needed.

**Comment by thewriteway** [comment]
Can you try this version: https://github.com/thewriteway/SidebarDiagnostics/releases/download/v3.7.5/SidebarDiagnostics_app-3.7.5.zip
